### PR TITLE
feat(Column/Bar/ColumnWithTrend-Chart): add custom coloring of elements

### PR DESCRIPTION
**Pull request by kleinju1017**
Added feature to customize the color of single Elements of Bar-, Column Chart and ColumnChartWithTrend. Extended API of measures to accept a function which defines the color and the elements of the measure.

- [x] add custom cell rendering to ColumnChart
- [x] extend api to customize single colors of cells 
- [x] add custom coloring to bar chart  

**Thank you for your contribution!** 👏

To get it merged faster, kindly review the checklist below:

## Pull Request Checklist
- [x] Reviewed the [Contributing Guidelines](https://github.com/SAP/ui5-webcomponents-react/blob/master/CONTRIBUTING.md)
    + Especially the [How to Contribute](https://github.com/SAP/ui5-webcomponents-react/blob/master/CONTRIBUTING.md#contribute-code) section 
- [x] [Correct commit message style](https://github.com/SAP/ui5-webcomponents-react/blob/master/docs/Guidelines.md#commit-message-style)

### CODE REVIEW

File: packages/charts/src/components/BarChart/BarChart.stories.mdx [hunk 1]
```diff
@@ -212,3 +212,35 @@ import { complexDataSet, secondaryDimensionDataSet, simpleDataSet } from '../../
...
+          accessor: 'users',
+          stackId: 'A'
+        },
+        {
+          accessor: 'sessions',
+          stackId: 'A'
+        },
+        {
+          accessor: 'volume',
+          highlightColor: (currVal, currMeasure, currData) => {
```
Comment (MarcusNotheis) [inline packages/charts/src/components/BarChart/BarChart.stories.mdx:235]:
```suggestion
          highlightColor: (value, measure, data) => {
```
Maybe this is slightly more readable for the demo?

File: packages/charts/src/components/ColumnChart/ColumnChart.stories.mdx [hunk 2]
```diff
@@ -192,3 +192,45 @@ import { complexDataSet, secondaryDimensionDataSet, simpleDataSet } from '../../
...
+        {
+          accessor: 'sessions',
+          label: 'Active Sessions',
+          formatter: (val) => `${val} sessions`,
+          hideDataLabel: true
+        },
+        {
+          accessor: 'volume',
+          label: 'Vol.',
+          highlightColor: (currVal) => {
```
Comment (MarcusNotheis) [inline packages/charts/src/components/ColumnChart/ColumnChart.stories.mdx:225]:
```suggestion
          highlightColor: (value) => {
```
Maybe this is slightly more readable for the demo?

File: packages/charts/src/hooks/useRenderHighlightedCells.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { getValueByDataKey } from 'recharts/lib/util/ChartUtils';
```
Comment (MarcusNotheis) [inline packages/charts/src/hooks/useRenderHighlightedCells.ts:1]: this file isn't used and can be deleted?

File: packages/charts/src/components/BarChart/BarChart.tsx [hunk 4]
```diff
@@ -66,6 +68,13 @@ interface MeasureConfig extends IChartMeasure {
    * @default undefined
    */
   stackId?: string;
+  /**
+   * Highlight color of defined elements
+   * @param value
+   * @param measure
+   * @param dataElement
```
Comment (MarcusNotheis) [inline packages/charts/src/components/BarChart/BarChart.tsx:75]: Maybe explain the params here? Same applies then for the `ColumnChart`

File: packages/charts/src/components/BarChart/BarChart.tsx [hunk 5]
```diff
@@ -107,6 +116,7 @@ export interface BarChartProps extends IChartBaseProps {
    * - `width`: bar width, defaults to `auto`
    * - `opacity`: bar opacity, defaults to `1`
    * - `stackId`: bars with the same stackId will be stacked
+   * - `highlightColor`: defines a custom color of a specific element. Overwrites code>color</code> of the element.
```
Comment (MarcusNotheis) [inline packages/charts/src/components/BarChart/BarChart.tsx:119]: Can you briefly outline the function signature here? Same applies for the `ColumnChart`

**Review by MarcusNotheis — CHANGES_REQUESTED** [review]

File: packages/charts/src/hooks/useRenderHighlightedCells.ts (hunk 3 above)
Comment (kleinju1017) [reply packages/charts/src/hooks/useRenderHighlightedCells.ts:1]: the function is used in line 9

**Review by MarcusNotheis — APPROVED** [review]

**Pull request merged**